parmed/charmm/psf.py

```diff
@@ -458,6 +458,10 @@ def load_parameters(self, parmset):
         parmset : :class:`CharmmParameterSet`
             List of all parameters
 
+        copy : bool
+            if False, parmset will not be copied. This will silently modify the references to types in the structure
```

Comment:
"This will **silently** modify the references to types in the structure" --> sounds dangerous.